charts/chartpress.yaml

```diff
@@ -0,0 +1,25 @@
+charts:
+  - name: renga
+    imagePrefix: rengahub/
+    repo:
+      git: SwissDataScienceCenter/helm-charts
+      published: https://swissdatasciencecenter.github.io/helm-charts
+    paths:
+      - .
+      - ../services
+    images:
+      apispec:
+        contextPath: ../services/apispec
+        dockerfilePath: ../services/apispec/Dockerfile
+        valuesPath: apispec.image
+        paths:
+          - ../services/apispec
+      singleuser:
+        contextPath: ../services/singleuser
+        buildArgs:
+          BASE_IMAGE: rengahub/minimal-notebook:e21fd4181aa2
+          JUPYTERHUB_VERSION: master
+        dockerfilePath: ../services/singleuser/Dockerfile
+        valuesPath: jupyterhub.notebook.image
```

Comment:
shouldn't this be `jupyterhub.singleuser.image`? The `jupyterhub.notebook.image` value isn't used anywhere in the jupyterhub chart.